tool/gravity/cli/plan.go

```diff
@@ -253,5 +256,9 @@ Clean up the node with 'gravity leave' and start the operation with either 'grav
 	noOperationStateNoClusterStateBanner = `no operation found.
 This usually means that the installation/join operation has failed to start or was not started.
 Start the operation with either 'gravity install' or 'gravity join'.
+`
+	noOperationPlanBanner = `%v has no plan and is invalid.
```

Comment:
Do you think that "and is invalid" is maybe unnecessary here? I'd just say "X has no plan".

Reply:
I'll remove the plan mention and will keep the invalid state - it will be reflected in the handling as well.